addons/interaction/CfgVehicles.hpp

```diff
@@ -251,6 +251,15 @@ class CfgVehicles {
                 statement = QUOTE([ARR_3(_player,_target,1)] call DFUNC(tapShoulder));
                 exceptions[] = {"isNotSwimming"};
             };
+            class ACE_OpenBackpack {
+                displayName = "$STR_ACTION_OPEN_BAG";
+                position = QUOTE(call DFUNC(getBackpackPos));
+                distance = 1.50;
```

Comment:
This is very short and can be extremely annoying to use.

Reply:
Seemed fine in testing, but I can bump it up. What about 3 meters?

Reply:
3 meters is what I use in AIMEE, personally I'd prefer that although other opinions are appreciated.